src/js/components/AppPageComponent.jsx

```diff
@@ -121,6 +122,21 @@ var AppPageComponent = React.createClass({
       this.onDeleteAppSuccess);
     AppsStore.removeListener(TasksEvents.DELETE_ERROR,
       this.onDeleteTaskError);
+    Mousetrap.unbind(["command+e", "ctrl+e"]);
+  },
+
+  componentDidMount: function () {
+    Mousetrap.bind(["command+e", "ctrl+e"], this.openEditModal.bind(this));
```

Comment:
In Chrome this has a function `Use selection in find`. Let's try to pick shortcuts that don't interfere with native behaviour when possible. how about just `e` when in the app's page?